src/encoded/schemas/changelogs/target.md

```diff
@@ -1,5 +1,9 @@
 ## Changelog for target.json
 
+### Minor changes since schema version 10
+
+* *investigated_as* enum changed to delete "control", "nucleotide modification", "other post-translational modification", and to add "Co-factor", "Cohesin", "DNA replication", "DNA repair", "RNA polymerase complex".
```

Comment:
I think that should be slightly reworded to be: "enum list was modified. The following terms were removed .. and the following terms were added ..."

Reply:
And "control" was not deleted from the list

Reply:
Fixed with the new commit.